packages/stack/src/components-page/README.md

```diff
@@ -0,0 +1,175 @@
+# Stack SDK Components Documentation
+
+The following documentation provides detailed information about various components and functionalities related to account settings, authentication, team interactions, and more within the Stack SDK. These components are essential for incorporating authentication and account management capabilities into applications using the Stack SDK.
+
+---
+
+## AccountSettings
+
+**Type**: Component
+
+**Description**: This component allows users to manage their account settings, including profile information, password, email verification, and multi-factor authentication (MFA).
+
+**Usage**:
+```jsx
+<AccountSettings fullPage={true} />
+```
+
+**Properties**:
+- `fullPage`: `boolean` - Determines if the settings should be displayed as a full page. Default is `false`.
+
+**Notes**: The component encapsulates several sections, each responsible for a specific part of the user settings, providing a comprehensive management interface.
```

Comment:
The notes are kinda redundant, would rather have a second paragraph in the description instead